src/index.js

```diff
@@ -0,0 +1,229 @@
+const inquirer = require("inquirer");
+
+const Db = require("./connection/connection");
+
+const {
+  generateDepartmentChoices,
+  generateRoleChoices,
+  generateEmployeeChoices,
+} = require("./helper/util");
+
+const {
+  menu,
+  addDepartment,
+  sectionToUpdate,
+  updateEmployeeName,
+  constructRoleQuestions,
+  constructDepartmentQuestions,
+  constructEmployeeQuestions,
+} = require("./questions/questions");
+
+//create a new instance of the Db class to use the start , stop and query methods
+const db = new Db({
+  host: process.envDB_HOST || "localhost",
+  user: process.envDB_USER || "root",
+  password: process.envDB_PASSWORD || "Password123!!",
+  database: process.envDB_NAME || "company_db",
+});
+
+const start = async () => {
+  let inProgress = true;
+
+  while (inProgress) {
+    // start the connection to the mysql database
+    await db.start();
+
+    // prompt a list of actions the user can perform
+    const { homeMenu } = await inquirer.prompt(menu);
+
+    // view list of all DEPARTMENT
+    if (homeMenu === "viewAllDepartments") {
+      const listOfDepartments = await db.query("SELECT * FROM department");
```

Comment:
Consider moving the action instructions into seperate files to keep this file clean. You could seperate actions into  `view.js`, `add.js` etc files so each function is easy to find for other developers. This avoids a long file with lots of functions.